proposed/http-client/http-client.md

```diff
@@ -43,10 +43,9 @@ interface HttpClient
     /**
      * Sends a PSR-7 request.
      *
-     * If a request is sent without any prior configuration, an exception MUST NOT be thrown
-     * when a response is recieved, no matter the HTTP status code.
+     * An exception MUST NOT be thrown when a response is recieved, no matter the HTTP status code.
```

Comment:
IMO an `HTTP/0.9` response should definitely throw an exception. In fact a HTTP/0.9 response is just a response body as defined in https://www.ietf.org/rfc/rfc1945.txt. If a HTTP/1.x response claims to be any version other than 1.x, it should definitely result in an exception.